### ISSUE TITLE
Language dropdown list does not have proper language names

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#35970]
# Steps to reproduce
1. checkout current core master
2. login to webUI  and browse to Settings, Personal, General
3. Select the dropdown to choose language

### Expected behaviour
The names of the languages should be written in script of each language.

### Actual behaviour
Many of the languages show the language code, like `ru` `fr` `de` 

### Server configuration
development using current core master

**Comment by phil-davis** [comment]
https://github.com/owncloud/core/commit/d39cc18bb0390ff229b30393741c9e624ad92a7a#r34543423
`settings/l10n/ru.js`
`settings/l10n/ru.json`

```
"__language_name__" : "Русский",
```
got deleted by the latest transifex commit in core.
(and similar for other languages)

Note: in `lib/l10n/ru.json` that line only got moved. So `lib/l10n` looks OK.
But I guess the webUI (and what other stuff?) is using `settings/l10n`
What is the reason for having both `lib/l10n` and `settings\l10n` ?

**Comment by phil-davis** [comment]
The CI of the Transifex commit to core master failed with exactly this problem:
https://drone.owncloud.com/owncloud/core/19911/859

```
  Scenario: change language                                                                       # /drone/src/tests/acceptance/features/webUIPersonalSettings/personalGeneralSettings.feature:13
    When the user changes the language to "Русский" using the webUI                               # WebUIPersonalGeneralSettingsContext::theUserChangesTheLanguageToUsingTheWebUI()
      Select option with value|text "Русский" not found.
```

It would be nice of Transifex would create a PR each night, let CI run on the PR, then only auto-merge if CI is green.

**Comment by phil-davis** [comment]
And a test run of CI also fails the same: https://drone.owncloud.com/owncloud/core/19914/791

**Comment by patrickjahns** [comment]
> It would be nice of Transifex would create a PR each night, let CI run on the PR, then only auto-merge if CI is green.

Thats currently not implemented - needs a change in the nightly jobs

**Comment by phil-davis** [comment]
Fix coming...

**Comment by phil-davis** [comment]
Should be fixed by #35973 - read that for details.

**Comment by phil-davis** [comment]
Note: this is not a bug in any existing real release. It is just something that got broken in the git repo and repaired before any release.
